schemas/core/core.graphql

```diff
@@ -188,11 +185,9 @@ type Team {
   updatedAt: DateTime
   deactivatedAt: DateTime
   description: String
-  createdTs: Int
-  deactivatedTs: Int
   createdByUser: User
   organization: Organization
-  members(pageNumber: Int, pageSize: Int, orderBy: TeamMemberUserOrderBy, desc: Boolean): [TeamMember]
+  members(pageNumber: Int, pageSize: Int, orderBy: UserUserOrderBy, desc: Boolean): [TeamMember]
```

Comment:
I think I may have introduced a but when generating the typename for this orderBy - it shouldn't have changed. (It gets generated in code, not hardcoded, hence me not catching this sooner)